tests/test_flake8_trio.py

```diff
@@ -50,6 +50,22 @@
 class ParseError(Exception): ...
 
 
+class ColumnAgnosticStatement(Statement):
+    def __eq__(self, other: object) -> bool:
+        assert isinstance(other, Statement)
+        return (
+            self.name == other.name
+            and self.lineno == other.lineno
+            and (
+                self.col_offset == other.col_offset
+                or -1 in (self.col_offset, other.col_offset)
+            )
+        )
+
+    def __hash__(self) -> int:
+        raise NotImplementedError
```

Comment:
Why not `super().__hash__()`?

Reply:
That could cause objects that are equal to have different hashes, which is Not Allowed :tm: 
https://docs.python.org/3/reference/datamodel.html#object.__hash__

Rereading it now, it turns out I misread that originally and I didn't have to complicate things as much as I did. I originally read it as `__eq__` objects had to have the same hash, and objects with same hash value had to be equal to eachother...

So I could implement `__hash__()` with `return hash(tuple(self.name, self.lineno))`